package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "mintbean-css-editor",
+  "version": "1.0.0",
+  "description": "Chrome extension that edits the CSS on your current page, made for Mintbean hackathon 2020-04-04",
+  "author": "Bryce McMath <bryce.j.mcmath@gmail.com>",
+  "license": "MIT",
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "prettier": "prettier \"src/**/*.{js,vue}\"",
+    "prettier:write": "npm run prettier -- --write",
+    "build": "cross-env NODE_ENV=production webpack --hide-modules",
+    "build:dev": "cross-env NODE_ENV=development webpack --hide-modules",
+    "build-zip": "node scripts/build-zip.js",
+    "watch": "npm run build -- --watch",
+    "watch:dev": "cross-env HMR=true npm run build:dev -- --watch"
+  },
+  "husky": {
+    "hooks": {}
+  },
+  "dependencies": {
+    "axios": "^0.19.0",
+    "emmet": "git+https://github.com/cloud9ide/emmet-core.git#41973fcc70392864c7a469cf5dcd875b88b93d4a",
```

Comment:
are you using emmet in your editor?
if not, you can take emmet out of dependencies and drop it in devdependencies.
although i see that this is from cloud9ide, so maybe you ARE using it as a plugin or something for Ace

Reply:
I am intentionally including emmet as a plugin for Ace. I was aiming to add html and js to the dropdown of options, but didn't have enough time